src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -692,5 +699,44 @@ public void BlockDataDownloadedIsCalled_ForFvBlockWithNullPointer_ResultShouldBe
             result.Should().BeFalse();
             initialChainTip.Block.Should().BeNull();
         }
+
+        /// <summary>
+        /// Issue 24 @ BlockDataDownloaded called for some blocks. Make sure CH.Block is not null and for the
+        /// first block true is returned and false for others.
+        /// </summary>
+        [Fact]
+        public void BlockDataDownloadedIsCalled_ForValidBlocksAfterFv_ResultShouldBeTrueForTHeFirstAndFalseForTheRest()
+        {
+            // Chain header tree setup. Initial chain has 4 headers with no blocks.
```

Comment:
In this test we don't have any requirement of not having blocks on the initial chain, therefore I would just remove it from the comment as well as from the code as it is not the default behaviour.